provider/lib/manager.js

```diff
@@ -113,8 +113,8 @@ module.exports = function (logger, triggerDB, redisClient) {
 	                if ( change.id != null){
 	                	doc_name = change.id; 
 	                }
-	                if ( change.doc != null && change.doc._rev != null ){
-	                	doc_revision= change.doc._rev; 
+	                if ( change.changes != null && change.changes[0].rev != null ){
```

Comment:
Please safe-guard against change.changes[0] being undefined:

if ( change.changes !== null && change.changes[0] != null && change.changes[0].rev != null ){

Otherwise you could get an error like this:

`Could not read 'rev' of undefined`

Thanks!